### PR TITLE
Alex's uninformed tinkering to try to get Point Decode/Import working

### PR DESCRIPTION
**Pull request by alextdavis**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
What is the reason for switching from "unknown" to None? This is the only question I have for this PR, everything else looks good!

**Review by ghost — APPROVED** [review]

File: importers/importer.py [hunk 1]
```diff
@@ -38,25 +49,30 @@ def get_id_of(table, name):
...
                 """INSERT INTO rooms (name, building_id, floor) VALUES (%s, %s, %s)""",
                 (point.room_name, building_id, try_cast_int(point.room_floor)))
-            room_id = get_id_of("rooms", point.room_name)
+            room_id = get_id_of("rooms", point.room_name, building_id=building_id)
 
         # {'room_id': room_id}
-        device_id = get_id_of("devices", point.device_name)
+        point.device_name = point.device_name or "UnID'd Device in " + point.building_name \
+                            + " " + point.room_name
+        device_id = get_id_of("devices", point.device_name, room_id=room_id)
```
Comment (grenche) [inline importers/importer.py:71]: Is there a reason you don't need to know the building in this case too? Or is that taken care of in the device_name

Comment (grenche) [reply importers/importer.py:71]: Same question goes for line 75.

**Review by grenche — COMMENTED** [review]
Overall, this looks good to me!

**Review by ethanacm — APPROVED** [review]

**Pull request merged**